src/util/server.c

```diff
@@ -484,6 +499,8 @@ int server_setup(const char *name, int flags,
         return ENOMEM;
     }
 
+    debug_from_responder = prg_name_is_responder(name);
```

Comment:
I would delegate this to service implementations.
IMO, name-based logic in generic framework feels error prone / easy to break.

Reply:
Or maybe we can have a default "false" set in server_setup() and then "true" in https://github.com/SSSD/sssd/blob/60353300d547f114ca1261e8c130ea72459b572a/src/responder/common/responder_common.c#L1265 (but we need to make sure every responder indeed use this)

Reply:
Made this change as suggested, `sss_process_init()` looks like it is called by each responder